iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java

```diff
@@ -96,28 +105,46 @@ protected void serializeImpl(DataOutputStream stream) throws IOException {
 
     stream.writeInt(regionGroupMap.size());
     for (Entry<String, List<TRegionReplicaSet>> entry : regionGroupMap.entrySet()) {
-      String storageGroup = entry.getKey();
+      String database = entry.getKey();
       List<TRegionReplicaSet> regionReplicaSets = entry.getValue();
-      BasicStructureSerDeUtil.write(storageGroup, stream);
+      BasicStructureSerDeUtil.write(database, stream);
       stream.writeInt(regionReplicaSets.size());
       regionReplicaSets.forEach(
           regionReplicaSet ->
               ThriftCommonsSerDeUtils.serializeTRegionReplicaSet(regionReplicaSet, stream));
     }
+
+    stream.writeInt(regionGroupCreateTimeMap.size());
+    for (Entry<TConsensusGroupId, Long> entry : regionGroupCreateTimeMap.entrySet()) {
+      TConsensusGroupId regionId = entry.getKey();
+      long createTime = entry.getValue();
+      ThriftCommonsSerDeUtils.serializeTConsensusGroupId(regionId, stream);
+      stream.writeLong(createTime);
+    }
   }
 
   @Override
   protected void deserializeImpl(ByteBuffer buffer) throws IOException {
-    int storageGroupNum = buffer.getInt();
-    for (int i = 0; i < storageGroupNum; i++) {
-      String storageGroup = BasicStructureSerDeUtil.readString(buffer);
-      regionGroupMap.put(storageGroup, new ArrayList<>());
+    int databaseNum = buffer.getInt();
+    for (int i = 0; i < databaseNum; i++) {
+      String database = BasicStructureSerDeUtil.readString(buffer);
+      regionGroupMap.put(database, new ArrayList<>());
 
       int regionReplicaSetNum = buffer.getInt();
       for (int j = 0; j < regionReplicaSetNum; j++) {
         TRegionReplicaSet regionReplicaSet =
             ThriftCommonsSerDeUtils.deserializeTRegionReplicaSet(buffer);
-        regionGroupMap.get(storageGroup).add(regionReplicaSet);
+        regionGroupMap.get(database).add(regionReplicaSet);
+      }
+    }
+
+    if (buffer.hasRemaining()) {
```

Comment:
We need to consider setting a default value (maybe the current time) when restarts with no data.

Reply:
Fixed